f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -180,6 +180,7 @@ def delete(self, resource, service, **kwargs):
         payload = kwargs.get("payload",
                              self._create_payload(resource, service))
         LOG.debug("%s payload is %s", self._resource, payload)
+        self.driver.annotate_service_members(service)
```

Comment:
would need to double check if this affects other resources' deletion process, since this is a common place(from a base class)